torch/onnx/symbolic_opset9.py

```diff
@@ -6302,6 +6302,22 @@ def is_pinned(g: jit_utils.GraphContext, self, device=None):
     return None
 
 
+@_onnx_symbolic("aten::resolve_conj")
+def resolve_conj(g: jit_utils.GraphContext, self: _C.Value) -> _C.Value:
+    # Noop in ONNX. This op is materializing conjugation w.r.t conjugate bit.
+    # It is only adjusting the internal representation in torch but not its "value".
+    # Hence from ONNX perspective this op does nothing.
+    return self
+
+
+@_onnx_symbolic("aten::resolve_neg")
+def resolve_neg(g: jit_utils.GraphContext, self: _C.Value) -> _C.Value:
```

Comment:
@justinchuby Do we need beartype in this function?

Reply:
We do actually. Good point